modules/apps/calendar/calendar-web-test/src/testFunctional/macros/CalendarNavigator.macro

```diff
@@ -30,6 +30,8 @@ definition {
 		var key_modalTitle = "${modalTitle}";
 
 		Click(locator1 = "Button#CLOSE_MODAL");
+
+		LiferayEvent.initializeLiferayEventLog();
```

Comment:
Same I said at https://github.com/john-co/liferay-portal/pull/293#pullrequestreview-408229050. Let's remove this commit and try to make this right.